### ISSUE TITLE
The program takes a little long to run

### ISSUE THREAD

**Issue by WrongMalik** [M-Jin-Gao/fast_depth_innovation#1]
Hello, thanks for this work. I try to run the code using "pretrained=True", and see that "t_gpu" is about 50ms.
But it takes too long to finish 1 epoch. One epoch need about 25 minutes.
I want to ask for the reason, maybe my knowledge is not enough

**Comment by M-Jin-Gao** [comment]
I am not sure why it runs so slowly on your device. One of the possible reasons is that the value of "stage" is set too large.

This network does not directly generate in-depth information. It aims to predict the residual between the depth information generated by the pre-trained network and the ground truth to improve the accuracy of the depth information.

If the "stage" of arg is equal to 2, it means that the network will calculate the residual twice. By inputting the RGB image and the depth information "d0" generated by the pre-trained network, it will generate the first residual "r0". Then, the depth information "d1" fed into the network at "t1" will be the residual "r0" plus the depth information "d0". After inputting RGB image and depth information "d1", it will generate a new residual "r1". The final depth information will be "d0" + "r0" + "r1"

By the way, please delete the lines of code in lines 283 and 284. args.stage = 4  and args.scale = 0.4
